src/encoded/static/components/PageTitle.js

```diff
@@ -43,7 +43,11 @@ const TITLE_PATHNAME_MAP = {
             return 'Search';
         },
         'calloutTitle' : function searchViewCalloutTitle(pathName, context, href, currentAction, schemas){
-            var thisTypeTitle = schemaTransforms.getSchemaTypeFromSearchContext(context, schemas);
+            const thisTypeTitle = (
+                schemaTransforms.getTitleForType(
+                    schemaTransforms.getSchemaTypeFromSearchContext(context), schemas
```

Comment:
We changed `getSchemaTypeFromSearchContext` in SPC to no longer prettify `type` (for utility + to match its functionality to docstring/title) and thus now need to wrap with `getTitleForType` to grab it from schemas.